src/app/upload_tasks/up.py

```
@@ -110,6 +122,7 @@ def upload_task(
     user: Dict[str, str] = None,
     store: TaskStorePyMysql =None,
     task_id: str = "",
+    check_cancel: Callable = None,
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)